bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "namespace": "ethereum",
   "name": "web3",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
Will this number get bumped automatically during the release process, or should it be `1.2.3`?

Reply:
It will update it to 1.2.3 automatically. As I know is the bower.json and package.js anyways no longer in usage I was just updating it for consistency reasons.